src/modules/hardware-wallet/README.md

```diff
@@ -0,0 +1,138 @@
+# Hardware-wallet
+
+This package implements communication to a hardware wallet device, and an interface for sending messages to the Radix Ledger App.
+
+Refer to the Radix APDU spec for detailed information on message formats:
+
+https://github.com/radixdlt/radixdlt-ledger-app/blob/master/APDUSPEC.md
+
+
+| Version | Device Support | 
+|---------|----------------|
+| 1.0.0   | Ledger Nano S  |
+
+
+
+## Usage
+
+Initializing Radix identity:
+
+    import { RadixHardwareWalletIdentity } from 'radixdlt'
+    import { app } from '@radixdlt/hardware-wallet'
```

Comment:
Not sure about calling it simply an `app`, you really can't tell what it is from that name

Reply:
renamed to ledgerApp